x-pack/plugin/eql/qa/rest/build.gradle

```diff
@@ -18,14 +18,18 @@ artifacts {
   restXpackTests(new File(projectDir, "src/yamlRestTest/resources/rest-api-spec/test"))
 }
 
+tasks.named('javaRestTest') {
+  usesDefaultDistribution()
```

Comment:
Is using the default distribution strictly required or can we get away with just installing the required plugins/modules for these tests?

Reply:
I don't know - I suspect it doesn't but there may be a few modules that it needs to work reliably.
The existing tests used the default distribution, and I wanted to limit this PR to a conversion of test frameworks.